lib/src/embedded/isolate_dispatcher.dart

```diff
@@ -43,6 +44,9 @@ class IsolateDispatcher {
   /// See https://github.com/sass/dart-sass/pull/2019
   final _isolatePool = Pool(sizeOf<IntPtr>() <= 4 ? 7 : 15);
 
+  /// Whether the stdin has been closed or not.
```

Comment:
Nit: better to say

```suggestion
  /// Whether [_channel] has been closed or not.
```

because this class doesn't know or care whether that's connected to stdin (and in fact it may not be in future implementations if, for example, we use this via WASM in the future).